hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -146,12 +146,24 @@ public class DatanodeConfiguration {
    */
   @Config(key = "block.delete.queue.limit",
       type = ConfigType.INT,
-      defaultValue = "1440",
+      defaultValue = "5",
       tags = {DATANODE},
       description = "The maximum number of block delete commands queued on " +
           " a datanode"
   )
-  private int blockDeleteQueueLimit = 60 * 24;
+  private int blockDeleteQueueLimit = 5;
```

Comment:
This seems like a very low value - any idea why the old value was such a specific value of 1440?

Reply:
Ah, ok. I wasn't aware of how the block delete flow worked, so 5 makes sense.